src/search/controllers/search.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { SearchService } from '../services/search.service';
+
+@Controller('search')
+export class SearchController {
+  constructor(private searchService: SearchService) {}
+
+  @Get('boards')
+  async searchBoardsAsHeadOrBody(@Query('searchQuery') searchQuery: string) {
```

Comment:
```suggestion
  async searchBoardsByHeadOrBody(@Query('searchQuery') searchQuery: string) {
```
이런 경우에 일반적으로 as 보다는 by를 믾이 씁니다

Reply:
넵